tests/v2/waku_archive/test_driver_postgres.nim

```diff
@@ -0,0 +1,184 @@
+{.used.}
+
+import
+  std/sequtils,
+  testutils/unittests,
+  chronos
+import
+  ../../../waku/v2/protocol/waku_archive,
+  ../../../waku/v2/config,
+  ../../../waku/v2/protocol/waku_archive/driver/postgres_driver,
+  ../../../waku/v2/protocol/waku_message,
+  ../testlib/common,
+  ../testlib/waku2
+
+proc defaultConf : WakuNodeConf =
+  return WakuNodeConf(
+    storeMessageDbUrl: "postgres://postgres:test123@localhost:5432/postgres",
+    listenAddress: ValidIpAddress.init("127.0.0.1"), rpcAddress: ValidIpAddress.init("127.0.0.1"), restAddress: ValidIpAddress.init("127.0.0.1"), metricsServerAddress: ValidIpAddress.init("127.0.0.1"))
```

Comment:
We should probably pull this from env? that depends on how/when we want to run postgres tests